cfgov/jinja2/v1/_includes/molecules/featured-menu-content.html

```diff
@@ -0,0 +1,43 @@
+{# ==========================================================================
+
+   featured_menu_content
+
+   ==========================================================================
+
+   Description:
+
+   Create a featured content molecule. See
+   [GHE]/flapjack/Modules-V1/wiki/Featured-Menu-Content
+
+   value: An object with the following options for value.
+
+   value.body:  Body text.
+
+   value.link: A tuple of link info, containing:
+
+   value.link.url:  A string for the URL of the link.
+
+   value.link.text: A string for the text of the link.
+
+   value.image: A tuple of image info, containing:
+
+   value.image.src: A string for the image source URI.
+
+   value.image.alt: A string for the image alt text.
+
+   ========================================================================== #}
+
+{% macro render( value ) %}
+<a href="{{ value.link.url }}">
+    <img src="{{ value.image.src }}"
+         alt="{{ value.image.alt }}"
+         height="150"
+         width="270" />
+    <p class="h4 u-mt15">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1665/commits/60b8eaf28eafee20484c08edd6d6aeb6fb37dce1